Lombiq.Hosting.Azure.ApplicationInsights/ApplicationInsightsOptions.cs

```diff
@@ -70,6 +70,14 @@ public class ApplicationInsightsOptions
     /// </summary>
     public bool EnableClientSideTracking { get; set; } = true;
 
+    /// <summary>
+    /// Gets or sets a value indicating whether local development is enabled where telemetry is sent to
+    /// Azure Application Insights from the local instance.
+    /// Will only work if local authentication is enabled in the properties of the Applications Insights resource
+    /// which should only be the case during development and testing.
```

Comment:
Keep to https://lombiq.atlassian.net/wiki/spaces/DEV/pages/28213305/General+code+conventions+and+development+practices#:~:text=For%20legibility%20try%20to%20limit%20lines%20of%20code%20to%20120%20characters%20(soft%20limit)%20or%20150%20at%20most%20(hard%20limit%2C%20code%20analyzers%20will%20warn%20about%20it%20too)%2C%20if%20breaking%20at%20120%20would%20hurt%20compactness%20or%20legibility.

Reply:
This isn't addressed.

Reply:
> This isn't addressed.

Oh I forgot to ask you about this, I don't really know what you meant by this because none of these lines exceed this rule the largest line is 114 characters

Reply:
They're shorter though. and the first two lines can be packed tighter, without going over the limit. The Rewrap extension will reformat as such.

Reply:
Understood, should be addressed in the most recent push